labs-ios-starter/Profiles/Model Controllers/ProfileController.swift

```diff
@@ -57,34 +66,42 @@ class ProfileController {
             
             defer {
                 DispatchQueue.main.async {
-                    completion()
+                    completion(nil)
                 }
             }
             
             if let error = error {
                 NSLog("Error getting all profiles: \(error)")
+                completion(ErrorHandler.UserAuthError.noConnection)
+                return
             }
             
             if let response = response as? HTTPURLResponse,
                 response.statusCode != 200 {
                 NSLog("Returned status code is not the expected 200. Instead it is \(response.statusCode)")
```

Comment:
Could be shortened to `Expected status code 200, instead returned \(response.statusCode)`

Reply:
nice touch